adminStuff/addNewTag.php

```diff
@@ -0,0 +1,38 @@
+<?php
+include('config/init.php');
+if(isset($_REQUEST['submitItem'])){
+    $errors1 = array();
+
+    if(!@$_REQUEST['tagName']){
+        $errors1['tagName'] = "*Required*";
+    }
+    if(sizeof($errors1) == 0){
+        echo addNewTagToExistingPost($_REQUEST['tagName'], $_REQUEST['blogPostId']);
```

Comment:
Note: I won't call this out each time just to save you from reading redundant comments, but the stuff from the file above applies to the other forms as well